File: StockMarketCharting/StockMarket.AccountAPI/Services/UserService.cs
using StockMarket.AccountAPI.Models;
using StockMarket.AccountAPI.Repositories;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StockMarket.AccountAPI.Services
{
    public class UserService : IUserService
    {
        UserRepository _repo = new UserRepository();

        public void AddUser(User item)
        {
            _repo.AddUser(item);
        }

        public User Login(string uname, string passwd)
        {
            return _repo.Login(uname, passwd);
        }
    }
}

File: StockMarketAngular/src/app/services/user.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Observable } from 'rxjs';
import { User } from '../models/user';

@Injectable({
  providedIn: 'root'
})
export class UserService {
  path: string = "http://localhost:59472/api/Account/";

  constructor(private http: HttpClient) { }

  public Login(uname: string, passwd: string): Observable<User>
  {
    return this.http.get<User>(this.path+"Login/"+uname+"/"+passwd);
  }

  public Signup(item: User)
  {
    return this.http.post(this.path+"AddUser/",item);
  }
}
